src/main/java/it/gov/pagopa/spontaneouspayment/controller/impl/ServicesController.java

```diff
@@ -38,4 +44,31 @@ public ResponseEntity<List<ServiceModelResponse>> getServices() {
                 HttpStatus.OK);
     }
 
+	@Override
+	public ResponseEntity<ServiceDetailModelResponse> createService(@Valid ServiceDetailModel serviceModel) {
+		// flip model to entity
+		Service serviceEntity = modelMapper.map(serviceModel, Service.class);
+		return new ResponseEntity<>(
+                modelMapper.map(servicesService.createService(serviceEntity), ServiceDetailModelResponse.class),
+                HttpStatus.CREATED);
+	}
+
+	@Override
+	public ResponseEntity<ServiceDetailModelResponse> updateService(@NotBlank String serviceId, @Valid ServiceDetailUpdModel serviceUpdateModel) {
+		// flip model to entity
+		Service serviceEntity = modelMapper.map(serviceUpdateModel, Service.class);
+		serviceEntity.setId(serviceId);
+		return new ResponseEntity<>(
+                modelMapper.map(servicesService.updateService(serviceEntity), ServiceDetailModelResponse.class),
+                HttpStatus.OK);
+	}
+
+	@Override
+	public ResponseEntity<String> deleteService(@NotBlank String serviceId) {
```

Comment:
This is not a Json return. Use Void instead
```suggestion
	public ResponseEntity<Void> deleteService(@NotBlank String serviceId) {
```

Reply:
[latest specification](https://www.rfc-editor.org/rfc/rfc8259#section-2:~:text=Here%20are%20three%20small%20JSON%20texts%20containing%20only%20values%3A%0A%0A%20%20%20%22Hello%20world!%22%0A%0A%20%20%2042%0A%0A%20%20%20true) admits string (with format "\w*") as valid JSON